CharlieBackend.AdminPanel/Utils/HttpUtil.cs

```diff
@@ -21,7 +21,9 @@ public class HttpUtil : IHttpUtil
                         IHttpContextAccessor httpContextAccessor,
                         IDataProtectionProvider provider)
         {
-            _client = new HttpClient()
+            HttpClientHandler handler = new HttpClientHandler();
+            handler.ServerCertificateCustomValidationCallback = (sender, cert, chain, sslPolicyErrors) => true;
+            _client = new HttpClient(handler)
```

Comment:
Usually we add this handy code locally when having ssl trouble. Maybe remove HttpClientHandler?

Reply:
Removed